src/redux/gas.ts

```diff
@@ -1,5 +1,6 @@
 import analytics from '@segment/analytics-react-native';
 import { captureException } from '@sentry/react-native';
+import BigNumber from 'bignumber.js';
```

Comment:
This should be `import { BigNumber } from '@ethersproject/bignumber';`

Reply:
this change would break `parsers/gas` types https://github.com/rainbow-me/rainbow/blob/develop/src/parsers/gas.ts#L1 so I didn't want to mess something up there

Reply:
biggest problem is the `utilities` file that uses `bignumber.js`

Reply:
ooph yikes - made a ticket here: https://linear.app/rainbow/issue/RNBW-2006/cleanup-bignumber-usage-in-gas-parsers-esp-related-to-l1-optimism-fee